frontend-site/src/modules/staticData.js

```diff
@@ -1,29 +1,51 @@
 // TODO: decide what stays here and what can be stored as a JSON file for the API to send
 export const utilitiesOffered = Object.freeze([
+  '3-D Printing',
   'Raspberry Pi',
   'Arduino',
+  'LUG API',
+  'Board Game Night',
   'Ice Cream',
-  '3-D Printing',
 ]);
 
 export const servicesOffered = Object.freeze([
   // TODO: fill in empty entries
+  {
+    name: '3D Printing',
+    image: '',
+    description: 'Becoming a LUG member brings many perks such as having easy access to our 3D printer located in the ACM/LUG office. Members can watch their creations come alive for minimal cost.',
+    url: 'https://lug.cs.uic.edu/3dprint/home',
+  },
   {
     name: 'Raspberry Pi',
     image: '',
-    description: '',
+    description: 'At the center of student creativity in computing is the infamous Rasberry Pi, and LUG is here to help students gain access to one. We offer Rasberry Pi rentals for any student interested.',
     url: '',
   },
   {
     name: 'Arduino',
     image: '',
-    description: '',
+    description: 'If your looking for a microcontroller but dont want to buy your own, LUG has you covered. In addition to the Rasberry Pi, LUG offers Arduino rentals.',
```

Comment:
Spelling/Grammar: "If **you're** looking for ... but **don't** want to buy your own..."